scripts/create_mac_app.py

```
@@ -0,0 +1,155 @@
+echo "✅ Installation complete!"
+echo ""
+echo "To run the app:"
+echo "1. Open Finder"
+echo "2. Go to Applications"
+echo "3. Double-click 'iMessage Exporter'"
+echo ""
+echo "Or run from Terminal:"
+echo "open ~/Applications/iMessage\\ Exporter.app"
+echo ""
+echo "Note: You may need to allow the app in System Preferences > Security & Privacy"
+"""
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)